assimilate.csh

```diff
@@ -156,9 +156,9 @@ echo "Brian's suggestions of checks:"
 echo "which mpiexec_mpt"
 which mpiexec_mpt
 echo "mpif90 -show"
-mpif90 -show
-echo "ldd ./filter.exe"
-ldd ${EXEROOT}/filter.exe
+mpif90 -show | sed -e 's# -#\n     -#g'
+echo "ldd ./filter" 
+ldd ${EXEROOT}/filter 
```

Comment:
I guess I don't remember when _filter_ became _filter.exe_, so I certainly don't care that it is back to _filter_


```
0[1446] cheyenne6:/<3>f.e21.FHIST_BGC.f09_025.CAM6assim.011/bld > ls -lL filter
-rwxr-xr-x 1 raeder p86850054 10793216 Mar  2 18:48 filter
```
so that's right ...

Reply:
"filter.exe" was a mistake I imported from Brian's suggestion and didn't notice before committing it.
It failed, so I've corrected it now.